src/main/java/frc/robot/components/MachineLearning.java

```diff
@@ -100,20 +29,20 @@ public MachineLearning(boolean record, String filename) {
         backRightEncoderPosition.clear();
 
         if(record) {
-            executeCmd("mkdir /paths");
-            executeCmd("chmod a=rwx /paths");
 
             try {
+                new File("/paths/.dsx").mkdirs();
                 File myObj = new File("/paths/" + filename);
 
-                if (myObj.createNewFile())  {
+                if (myObj.createNewFile()) {
```

Comment:
I like how the directory creation is done in Java now. Can the `mkdirs` call be done on `myObj`?